chaoscenter/graphql/server/pkg/chaoshub/handler/handler_test.go

```diff
@@ -281,7 +281,7 @@ func TestGetChartsData(t *testing.T) {
 			name:      "success: url is valid",
 			projectID: uuid.New().String(),
 			repoData: model.CloningInput{
-				Name:       uuid.New().String(),
+				Name:       "container-kill",
```

Comment:
Why did you change this line? IMO, It's not relevant to your PR.

Reply:
Yep, that's true. The issue was causing the chaos-hub tests to fail because the generated UUID was unique every time, resulting in the file being absent and triggering an error, indicating 'file not found'